rootfs/mkrootfs.sh

```diff
@@ -0,0 +1,25 @@
+# From https://github.com/virt-do/lumper/blob/main/rootfs/mkrootfs.sh
+
+curl -O https://dl-cdn.alpinelinux.org/alpine/v3.14/releases/x86_64/alpine-minirootfs-3.14.2-x86_64.tar.gz
+
+mkdir alpine-minirootfs
+tar xf alpine-minirootfs-3.14.2-x86_64.tar.gz -C alpine-minirootfs
+rm alpine-minirootfs-3.14.2-x86_64.tar.gz
+
+pushd alpine-minirootfs
+cat > init <<EOF
+#! /bin/sh
+#
+# /init executable file in the initramfs 
+#
+mount -t devtmpfs dev /dev
+mount -t proc proc /proc
+mount -t sysfs sysfs /sys
+ip link set up dev lo
+
+exec /sbin/getty -n -l /bin/sh 115200 /dev/console
```

Comment:
This would only give us a shell, so I eventually expect that this is where we'd either pull the container image and then run it, or simply run it (`--offline`).

Reply:
I was thinking about replacing this by calling the kaps binary.

Reply:
> I was thinking about replacing this by calling the kaps binary.

Correct.